.pipelines/stages/jobs/steps/nuget-win-step.yml

```diff
@@ -15,6 +19,18 @@ steps:
     FolderPath: '$(Build.Repository.LocalPath)\src\csharp\bin\Release\'
     DisplayName: 'ESRP - Sign C# dlls'
     Pattern: '*OnnxRuntimeGenAI*.dll'
+- powershell: |
```

Comment:
Put it in the image.
Or use "dotnet" command instead.

Reply:
Will install nuget in image in order to keep consistence with other nuget pkg release script.

Reply:
Put the nuget install in image build